docker/Dockerfile.faux

```diff
@@ -45,6 +46,28 @@ COPY subset/bacnet/bacnetTests/src/main/resources/Faux*.json tmp/
 COPY --from=java_build /root/bacnet4j/bacnet4j-1.0-SNAPSHOT-all.jar bacnetTests/libs/
 RUN cd bacnetTests && ./gradlew build
 
+#Telnet dependancy
+RUN $AG update && $AG install -y telnetd
+RUN $AG update && $AG install xinetd
+RUN service xinetd start
+
+#SSH dependancy
+COPY subset/security/ssh.config ssh.config
+RUN cat ssh.config >> /etc/ssh/sshd_config
+
+#HTTP/HTTPS dependancy
+RUN $AG update && $AG install nginx
+COPY /subset/security/nginxpass.conf /root/nginx/
+COPY /subset/security/nginxfail.conf /root/nginx/
+COPY /subset/security/nginx-site /var/www/nginx-site
+
+# SSH login fix. Otherwise user is kicked off after login
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+ENV NOTVISIBLE "in users profile"
```

Comment:
What is NOTVISIBLE for?  This is not-obvious, so please add explanatory comment.

Reply:
line turned out to not be necessary, been removed